src/main/java/org/graylog2/gelfclient/GelfConfiguration.java

```diff
@@ -147,6 +151,66 @@ public GelfConfiguration queueSize(final int size) {
     }
 
     /**
+     * Check if TLS option for the transport is enabled.
+     *
+     * @return true if TLS is enabled, false if disabled
+     */
+    public boolean isTls() {
```

Comment:
Although I'm well aware of the naming scheme of methods in this class. Maybe we want to update them as well in the future.